ibtk/include/ibtk/LDataManager.h

```diff
@@ -86,7 +86,90 @@ namespace IBTK
  * for all inter-processor communications.  All access to instantiated
  * LDataManager objects is via the static method getManager().
  *
+ * Each Lagrangian point is associated with an LNode object which provides an interface between the Lagrangian and PETSc
+ * ordering as well as the data storage for forces and objects assosciated with the point. From each LNode object, we
+ * can retrieve the Lagrangian index as well as any associated node data. The LNode objects for each processor are
+ * contained in an LMesh object. The LMesh object contains two vector of LNodes: one containing just the local LNode
+ * objects, and another containing 'ghost' LNodes. The 'ghost' LNodes are Lagrangian points located on other processors
```

Comment:
```suggestion
 * objects, and another containing 'ghost' LNodes. The 'ghost' LNodes are Lagrangian points assigned to other processors
```
?

Reply:
(If we don't already do so, maybe we also need to talk somewhere about how nodes are currently assigned to processors.)

Reply:
Do you want to say something more than "Nodes are assigned to processors based on the distribution of patches"?

Reply:
I think that is about it. Could maybe say "currently based on"...?